README.md

```diff
@@ -23,11 +23,11 @@ circuit.add(
     <RedLed />
   </board>
 )
-`)
+`);
 
-await circuitWebWorker.renderUntilSettled()
+await circuitWebWorker.renderUntilSettled();
 
-const circuitJson = await circuitWebWorker.getCircuitJson()
+const circuitJson = await circuitWebWorker.getCircuitJson();
 ```
```

Comment:
use `git checkout origin/main -- README.md` on files you didn't mean to change

check your work by looking at files changed and make sure you're not submitting unnecessary code